functions.php

```diff
@@ -154,9 +154,9 @@ function twentytwenty_register_scripts() {
 			wp_enqueue_script( 'comment-reply' );
 		}
 
-		$js_dependencies = array( 'jquery' );
+		$js_dependencies = array( 'wp-polyfill', 'wp-dom-ready' );
```

Comment:
I might be wrong but polyfill and dom-ready are available only in WP v5.*, everything below that will cause issues.